driver-core/src/test/java/com/datastax/driver/core/querybuilder/QueryBuilderTest.java

```diff
@@ -922,6 +922,7 @@ public void should_return_object_at_ith_index() {
     }
 
     @Test(groups = "unit")
+    @CassandraVersion(major = 2.1)
```

Comment:
Is the `@CassandraVersion` annotation necessary here?  These are just unit tests so the cassandra configuration shouldn't matter.